tests/manual/test_supersession_query.py

```
@@ -0,0 +1,126 @@
+    embedder=embedder,
+    knowledge_graph=knowledge_graph,
+)
+print("   ✓ Registry initialized")
+
+# Test 1: browse_entities to find regulations
+print("\n[3] Step 1: Browse regulations (testing browse_entities tool)...")
+browse_tool = registry.get_tool('browse_entities')
+result = browse_tool.execute(entity_type="standard", limit=10)
+
+if result.success:
+    print(f"   ✅ browse_entities SUCCESS")
```

Ruff (0.14.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)